plugin/src/test/java/org/opensearch/ml/rest/MLCommonsRestTestCase.java

```diff
@@ -283,7 +284,16 @@ protected void validateStats(
         }
         assertEquals(expectedTotalFailureCount, totalFailureCount);
         assertEquals(expectedTotalAlgoFailureCount, totalAlgoFailureCount);
-        assertEquals(expectedTotalRequestCount, totalRequestCount);
+        // ToDo: this line makes this test flaky as other tests makes the request count not predictable
+        // assertEquals(expectedTotalRequestCount, totalRequestCount);
```

Comment:
I think it's ok to remove this line, multiple tests against same cluster may have different total requests based on their execution sequence.